src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -50,13 +50,6 @@ public function __construct(?string $globalFailureReceiverName, ServiceProviderI
         parent::__construct();
     }
 
-    protected function getReceiverName(): string
-    {
-        trigger_deprecation('symfony/messenger', '5.3', 'The method "%s()" is deprecated, use getGlobalFailureReceiverName() instead.', __METHOD__);
-
-        return $this->globalFailureReceiverName;
-    }
```

Comment:
unrelated change, but make the bridge deprec-free. The class is internal so no need for a changelog entry (we didn't even add one when deprecating)